newIDE/app/src/GameDashboard/GameDetailsDialog.js

```diff
@@ -49,7 +49,7 @@ import { useResponsiveWindowWidth } from '../UI/Reponsive/ResponsiveWindowMeasur
 export type GamesDetailsTab =
   | 'details'
   | 'builds'
-  | 'feedbacks'
+  | 'feedback'
```

Comment:
haha, I remember changing feedback into feedbacks, looks like it was a bad move!

Reply:
Well it's to have the same cardinality as for the tab 😬